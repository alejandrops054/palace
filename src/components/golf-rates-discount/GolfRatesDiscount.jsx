/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from "react";
import { Card, CardBody, CardTitle } from 'reactstrap';
import { SimpleTable } from '../index'

function GolfRatesDiscount() {
    const columns = [
        {
            title: 'Golf Round',
            dataIndex: 'golfround',
            key: 'golfround',
            width: "1%",
        },
        {
            title: 'Hotel',
            dataIndex: 'hotel',
            key: 'hotel',
            width: "1%",
        },
        {
            title: 'Amount',
            dataIndex: 'amount',
            key: 'amount',
            width: "1%",
        },
        {
            title: 'Use Year',
            dataIndex: 'useyear',
            key: 'useyear',
            width: "1%",
        },
        {
            title: 'Use Date From',
            dataIndex: 'usedatefrom',
            key: 'usedatefrom',
            width: "1%",
        },
        {
            title: 'Use Date To',
            dataIndex: 'usedateto',
            key: 'usedateto',
            width: "1%",
        },
        {
            title: 'Print Date',
            dataIndex: 'printdate',
            key: 'printdate',
            width: "1%",
        },

    ];
    const data = [
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
        { golfround: '', hotel: '', amount: '', useyear: '', usedatefrom: '', usedateto: '', printdate: '' },
    ];
    return (
        <>
            <Card>
                <CardBody>
                    <CardTitle tag="h5">Golf Rates 50% Discount</CardTitle>
                    <SimpleTable columns={columns} data={data} type="table" />
                </CardBody>
            </Card>
        </>
    );
}

export default GolfRatesDiscount;
