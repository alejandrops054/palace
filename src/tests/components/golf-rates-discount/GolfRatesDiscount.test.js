/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import GolfRatesDiscount from '../../../components/golf-rates-discount/GolfRatesDiscount'

describe('GolfRatesDiscount component', () => {
    test('debe mostrar snapshot', () => {

        const wrapper = shallow(<GolfRatesDiscount />)
        expect(wrapper).toMatchSnapshot();
    })

})