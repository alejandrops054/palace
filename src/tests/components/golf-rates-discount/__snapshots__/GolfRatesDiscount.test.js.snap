// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GolfRatesDiscount component debe mostrar snapshot 1`] = `
<Fragment>
  <Card
    tag="div"
  >
    <CardBody
      tag="div"
    >
      <CardTitle
        tag="h5"
      >
        Golf Rates 50% Discount
      </CardTitle>
      <SimpleTable
        columns={
          Array [
            Object {
              "dataIndex": "golfround",
              "key": "golfround",
              "title": "Golf Round",
              "width": "1%",
            },
            Object {
              "dataIndex": "hotel",
              "key": "hotel",
              "title": "Hotel",
              "width": "1%",
            },
            Object {
              "dataIndex": "amount",
              "key": "amount",
              "title": "Amount",
              "width": "1%",
            },
            Object {
              "dataIndex": "useyear",
              "key": "useyear",
              "title": "Use Year",
              "width": "1%",
            },
            Object {
              "dataIndex": "usedatefrom",
              "key": "usedatefrom",
              "title": "Use Date From",
              "width": "1%",
            },
            Object {
              "dataIndex": "usedateto",
              "key": "usedateto",
              "title": "Use Date To",
              "width": "1%",
            },
            Object {
              "dataIndex": "printdate",
              "key": "printdate",
              "title": "Print Date",
              "width": "1%",
            },
          ]
        }
        data={
          Array [
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
            Object {
              "amount": "",
              "golfround": "",
              "hotel": "",
              "printdate": "",
              "usedatefrom": "",
              "usedateto": "",
              "useyear": "",
            },
          ]
        }
        type="table"
      />
    </CardBody>
  </Card>
</Fragment>
`;
